apps/prairielearn/assets/stylesheets/questionsTable.css

```diff
@@ -2,16 +2,17 @@
 @import url('bootstrap-table/dist/extensions/sticky-header/bootstrap-table-sticky-header.min.css');
 @import url('bootstrap-table/dist/extensions/filter-control/bootstrap-table-filter-control.min.css');
 
-.sticky-column {
+.table > :not(caption) > * > .sticky-column {
   position: sticky;
   left: 0;
-  background: white;
+  background-color: var(--bs-body-bg, white);
   background-clip: padding-box;
   box-shadow: inset -1px 0 #dee2e6;
 }
 
-.table-hover tbody tr:hover td.sticky-column {
-  color: #212529;
+.table.table-hover > tbody > tr:hover > td.sticky-column {
+  color: var(--bs-table-hover-color, #212529);
+  /* This cannot be transparent, to ensure the cell shows on top of others. TODO: handle a non-white background by combining --bs-body-bg with --bs-table-hover-bg. */
   background-color: #efefef;
```

Comment:
Ideally this color would be computed by applying the non-opaque version of --bs-table-hover-bg (using its opacity as the weight) with --bs-body-bg. There are some recent updates to CSS that could be used for this calculation (e.g., [color-mix](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/color-mix) or [relative colors](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_colors/Relative_colors)). I tried to play a bit with it but wasn't able to come up with anything simple, so I kept the current fixed color for now. A future PR (likely before or during the implementation of #3814) could take care of this.